backend/src/main/resources/db/migration/V1.11__file_sharing.sql

```diff
@@ -2,7 +2,8 @@ create table files (
     id uuid primary key,
     requested_at timestamp not null,
     uploader text not null,
-    group_id integer not null references groups_table(group_id)
+    group_id integer not null references groups_table(group_id),
+    public_at timestamp
```

Comment:
Or something else? `published_at`? `made_public_at`? `became_public_at`?

Reply:
Great, then let's go with `published_at`!

Reply:
I will make a commit to change it 👌

Reply:
https://github.com/loculus-project/loculus/pull/3983/commits/d93ceabca2e87a14a54c6f53e62274303499a8f1